src/main/java/edu/harvard/iq/dataverse/ingest/tabulardata/impl/plugins/dta/DTAFileReaderSpi.java

```diff
@@ -131,6 +131,7 @@ public boolean canDecodeInput(Object source) throws IOException {
 
     @Override
     public boolean canDecodeInput(BufferedInputStream stream) throws IOException {
+    	//who closes this stream?
```

Comment:
guessing this method could/should close it but I didn't see where it was getting called.

Reply:
We are not calling these methods at this time. They are part of a builtin infrastructure for automatically matching ingest plugins to specific file types... We are not using it now, but we may go back to it.  
It would be the responsibility of the code that calls these methods to close the streams.